Example/IGListKitExamples/SectionControllers/DemoSectionController.swift

```diff
@@ -53,7 +53,17 @@ class DemoSectionController: IGListSectionController, IGListSectionType {
     }
 
     func didSelectItem(at index: Int) {
-        if let controller = object?.controllerClass.init() {
+        if object?.name == "Storyboard" {
+            let storyboard = UIStoryboard(name: "Demo", bundle: nil)
+            let controller = storyboard.instantiateViewController(withIdentifier: "demo") as! StoryboardViewController
+            controller.title = object?.name
+            viewController?.navigationController?.pushViewController(controller, animated: true)
+        } else if object?.name == "Single Section Storyboard" {
+            let storyboard = UIStoryboard(name: "Demo", bundle: nil)
+            let controller = storyboard.instantiateViewController(withIdentifier: "singleSectionDemo") as! SingleSectionStoryboardViewController
+            controller.title = object?.name
+            viewController?.navigationController?.pushViewController(controller, animated: true)
```

Comment:
I thought about this for some time before putting it this way. Because it really makes no sense to give the cell identifier the same string as VC title. So if we combine this two together, I think we need a dictionary for title->identifier, right?